config/config_test.go

```diff
@@ -38,6 +38,7 @@ func TestMarshalUnmarshal(t *testing.T) {
 		Server: Server{
 			Address: "foo-test",
 			Port:    321,
+			StaticContentRootDirectory: "/tmp",
```

Comment:
This isn't testing what we need to test. We need to test going FROM YAML to data.

So you need to test that this parses correctly:

```
server:
   static_content_root_directory: /tmp
```

Reply:
Below is marshaled and unmarshaled before the test.

Reply:
yeah, but for all you know, the marshalling turned the propery name into "staticcontentrootdirectory" which then gets properly read in.

We need to test that that "key;omitempty" is actually doing what we expect - that the yaml actually is the one with the underscores.

Reply:
Just uploaded a new test, not sure if is the right way to test this.